config.ini

```diff
@@ -63,6 +63,7 @@ Enabled = True
 EnabledSubs = ss
 ReplaceLimit =
 FatigueSwitch = False
+UseBuckets = false
```

Comment:
Please change `false` to `False` (`ConfigReader` can deal with `false`, but to keep in-line with Python standards). If you could also update `config_detailed.ini` with this new parameter and add a description of what it does that'd be great.